index.html

```diff
@@ -72,7 +72,7 @@
 
             <div class="button_container">
               <p class="button_group_text site_font">Learn more about skydiving. </p>
-                <img class="skydiver_image" src="./assets/skydiver3.png">
+                <img class="skydiver_image adjust" src="./assets/skydiver3.png">
```

Comment:
Not a big fan of the class name "adjust" -- would prefer something like "shrink", "grow", "center", "moveLeft" etc. Something describing the adjustment